modules/apigee_edge_teams/tests/src/Functional/UITest.php

```diff
@@ -0,0 +1,375 @@
+<?php
+
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License version 2 as published by the
+ * Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+namespace Drupal\Tests\apigee_edge_teams\Functional;
+
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Url;
+use Drupal\Tests\apigee_edge\Traits\EntityUtilsTrait;
+use Drupal\Tests\field_ui\Traits\FieldUiTestTrait;
+
+/**
+ * Team and team app entity UI tests.
+ *
+ * @group apigee_edge
+ * @group apigee_edge_teams
+ */
+class UITest extends ApigeeEdgeTeamsFunctionalTestBase {
+
+  use EntityUtilsTrait;
+  use FieldUiTestTrait;
+
+  /**
+   * The team entity storage.
+   *
+   * @var \Drupal\apigee_edge_teams\Entity\Storage\TeamStorageInterface
+   */
+  protected $teamStorage;
+
+  /**
+   * Default user.
+   *
+   * @var \Drupal\user\UserInterface
+   */
+  protected $account;
+
+  /**
+   * Other user to test team membership related UIs.
+   *
+   * @var \Drupal\user\UserInterface
+   */
+  protected $otherAccount;
+
+  /**
+   * Team entity to test.
+   *
+   * @var \Drupal\apigee_edge_teams\Entity\TeamInterface
+   */
+  protected $team;
+
+  /**
+   * Product to test team app entity.
+   *
+   * @var \Drupal\apigee_edge\Entity\ApiProductInterface
+   */
+  protected $product;
+
+  /**
+   * Custom field properties and data of teams and team apps.
+   *
+   * @var array
+   */
+  protected $fields;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $this->teamStorage = $this->container->get('entity_type.manager')->getStorage('team');
+
+    $this->installExtraModules(['block']);
+    $this->drupalPlaceBlock('system_breadcrumb_block');
+    $this->drupalPlaceBlock('local_tasks_block');
+    $this->drupalPlaceBlock('local_actions_block');
+
+    $this->product = $this->createProduct();
+    $this->account = $this->createAccount([
+      'administer team',
+    ]);
+    $this->otherAccount = $this->createAccount();
+
+    $this->addFieldsToEntities();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function tearDown() {
+    try {
+      if ($this->account !== NULL) {
```

Comment:
Move the conditional outside of the try catch, only run the code if within the try catch if the condition is true.